modules/ROOT/pages/dataweave-runtime-functions.adoc

```diff
@@ -20,32 +49,109 @@ The `p` function provides access to properties, whether these are:
 
 For more on configuring properties and how they are handled, see xref:configuring-properties.adoc[Configure Properties].
 
-The following example will log the value of the property `some.property`.
+The following example logs the value of the property `http.port`.
 
-.Example: Property Function Usage
+.Example: Property Function Usage for Mule 4.1.4 and later
 [source,xml,linenums]
 ----
 <flow name="simple">
-  <logger message="#[p('some.property')]"/>
+  <logger level="INFO" doc:name="Logger"
+    message="#[Mule::p('http.port')]"/>
+</flow>
+----
+
+Note that you can also use the function when defining a DataWeave variable
+or function. This example uses the `p` function in the `myVar` definition.
+
+[source,xml,linenums]
+----
+<flow name="simple">
+  <logger level="INFO" doc:name="Logger"
+    message="#[var myVar = Mule::p('http.port') --- myVar]"/>
+</flow>
+----
+
+.Example: Property Function Usage for Mule 4.1.3 and Earlier
+[source,xml,linenums]
+----
+<flow name="simple">
+  <logger message="#[p('http.port')]"/>
+</flow>
+----
+
+[[function_lookup]]
+=== Execute a Flow (lookup Function)
+
+Similar to the Flow Reference component, the `lookup` function enables you to
+execute another flow within your app and to retrieve the resulting payload. It
+takes the flow's name and an input payload as parameters. For example, `lookup("anotherFlow", payload)` executes a flow named `anotherFlow`.
+
+The function executes the specified flow using the current attributes,
+variables, and any error.
+
+//TODO: VERIFY BELOW, AND CLARIFY BEFORE REPUBLISHING
+//It only replaces the payload for the first parameter, which must be a Java type.
+
+==== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name | Description
+| `flowName` | A string that identifies the target flow.
+| `payload` | The payload to send to the target flow, which can be any (`Any`) type.
```

Comment:
check this, has to be java

Reply:
Hey, you know, I'm looking at the function sig in Mule now: fun lookup(flowName: String, payload: Any, timeoutMillis: Number = 2000) =
    dw::mule::internal::Bindings::callFunction("", "lookup", [flowName, payload, timeoutMillis])